nucleus/settings.py

```diff
@@ -84,9 +84,9 @@ def git_repo_path(*args):
 MIDDLEWARE = (
     "allow_cidr.middleware.AllowCIDRMiddleware",
     "django.middleware.security.SecurityMiddleware",
+    "whitenoise.middleware.WhiteNoiseMiddleware",
     "nucleus.base.middleware.HostnameMiddleware",
     "django.middleware.http.ConditionalGetMiddleware",
-    "whitenoise.middleware.WhiteNoiseMiddleware",
```

Comment:
Just moving this up in line with what the Whitenoise docs recommend. I don't see that it'd be a problem to move it